solr/core/src/java/org/apache/solr/core/PluginBag.java

```diff
@@ -63,17 +64,20 @@
   private final Class<T> klass;
   private SolrCore core;
   private final SolrConfig.SolrPluginInfo meta;
-  private final ApiBag apiBag;
-  private final ResourceConfig jerseyResources;
 
   public static class JerseyMetricsLookupRegistry
       extends HashMap<Class<? extends JerseyResource>, RequestHandlerBase> {}
 
+  // Used for v2 API lookup and processing
+  private static final boolean IS_V2_ENABLED =
```

Comment:
oh goody, we duplicated this static variable ;-)

Reply:
I ended up dropping the duplicated static variable.